src/compiler/parser.ts

```diff
@@ -6468,7 +6468,7 @@ namespace ts {
 
                 function parseTagComments(indent: number) {
                     const comments: string[] = [];
-                    let state = JSDocState.SawAsterisk;

```

Comment:
Makes sense. There was a similar bug in the start state for the main jsdoc parsing, so it makes sense that you ran into it here too.